lab-jason/lib/joke.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+function tellJoke(filename, cb){
+  fs.readFile(`./assets/${filename}`, (err, buff) => {
+    if (err) {
+      cb(err);
+    }
+    if(cb) { cb(null, buff.toString('hex', 0, 8)); }
+  });
+}
+
+tellJoke('one.txt',
+  () => tellJoke('two.txt',
+    () => tellJoke('three.txt', (err) => {
+      if (err) throw err;
+    })
+  )
+);
+
+exports.readJoke = function(file1, file2, file3, cb) {
+  var datSmash = '';
+  tellJoke(file1, (err, data1) => {
+    if (err) { cb(err); }
```

Comment:
Need to `return` in here (either on a second line, or just `return cb(err)`, to make sure we don't continue.

Unless you mean to continue, in which case calling `cb(err)` is probably not what you want to do (rather than collecting an array of the errors, for example).